No need to merge, just want to say thank you 👍

mikhail-m1 opened the pull request:
hey, I have finally launched the example on my board, I have esp32s3 so and no power pin on my board, so I had to make some changes and I also added HTTP server to the example and deleted some useless files from my point of view :). I can create a proper PR if you want, but maybe we need to send PR to the original author. 

Anyway thanks for your work.

MathiasPius commented:
@mikhail-m1 I somehow completely missed your pull request, sorry!

Thank you so much for these changes, I was just thinking about adding an http server example myself.

Can I get you to move your changed example into a separate "example" project, just to maintain both the plain esp32 example as well as a separate esp32s3 one?

mikhail-m1 commented:
hey, some time passed since the PR was created and I significantly changed the library interface in my fork. I replaced new function by builder pattern, so more flexible way to configure is supported, added geters and changed setters, you can take a look. I wasn't following you changes, I think I need to look at your fork too and think what to do.

MathiasPius commented:
That's completely understandable. Do you mind if I manually try to backport the web-example?

mikhail-m1 commented:
sure, you can backport it if you want, or if I have time I will change the PR to contain only this change
